# StartAuction.srv - Service to start the auction
---
bool success
string message